src/playback/internal/au3/au3player.cpp

```diff
@@ -220,7 +220,7 @@ muse::Ret Au3Player::doPlayTracks(TrackList& trackList, double startTime, double
     return success ? muse::make_ok() : muse::make_ret(muse::Ret::Code::InternalError);
 }
 
-void Au3Player::seek(const muse::secs_t newPosition)
+void Au3Player::seek(const muse::secs_t newPosition, bool applyIfPlaying)
 {
     LOGD() << "newPosition: " << newPosition;
 
```

Comment:
Is this comment still relevant?

Reply:
yes, useful

Reply:
My bad, the comment in question is not shown on the PR page. I was talking about 

    //! TODO At the moment not work
    //! there probably should be a different implementation
    
    It does work now, right?

Reply:
ah, yes, no relevant, I think